src/procedures/procedures.11tydata.yaml

```diff
@@ -246,7 +246,8 @@ procedureMoveDataDisk:
       image: ha-config-storage.png
       content: |
         <ul>
-          <li>
+          <li class="warning"><b>CM4 Lite</b>: If you have a Raspberry&nbsp;Compute&nbsp;Module&nbsp;4&nbsp;Lite: As there is no eMMC&nbsp;flash, you can skip this and the following steps.</li>
+          <li class="info"><b>Regular CM4 modules</b>: This <i>moving your data</i> procedure is required, even if your module is new.<li>
```

Comment:
```suggestion
          <li class="info"><b>Regular CM4 modules</b>: This <i>moving your data</i> procedure is required for making use of your SSD, even if your module is new.<li>
```

Reply:
Just as an idea to make it clearer (because ofc. the system itself functions without 😁).
Else: Great work @c0ffeeca7 🙏